instrumentation/net/http/otelhttp/example/server/server.go

```diff
@@ -53,6 +58,17 @@ func initTracer() (*sdktrace.TracerProvider, error) {
 	return tp, err
 }
 
+func initMeter() (*sdkmetric.MeterProvider, error) {
+	enc := json.NewEncoder(os.Stdout)
+	exp, err := stdoutmetric.New(stdoutmetric.WithEncoder(enc))
+	if err != nil {
+		return nil, err
+	}
+
+	mp := sdkmetric.NewMeterProvider(sdkmetric.WithReader(sdkmetric.NewPeriodicReader(exp)))
```

Comment:
This needs to be [set as the global meter provider](https://pkg.go.dev/go.opentelemetry.io/otel/metric@v0.33.0/internal/global#SetMeterProvider) or in the `main` loop in needs to be passed directly to the instrumentation.

It doesn't look like this is currently used. Have you tested this example to produce the telemetry you are expecting?

Reply:
Sorry, I have tested in the first time, and saw the result is , so I thought the demo is right.
```
{"Resource":[{"Key":"service.name","Value":{"Type":"STRING","Value":"unknown_service:server"}},{"Key":"telemetry.sdk.language","Value":{"Type":"STRING","Value":"go"}},{"Key":"telemetry.sdk.name","Value":{"Type":"STRING","Value":"opentelemetry"}},{"Key":"telemetry.sdk.version","Value":{"Type":"STRING","Value":"1.11.1"}}],"ScopeMetrics":[]}
{"Resource":[{"Key":"service.name","Value":{"Type":"STRING","Value":"unknown_service:server"}},{"Key":"telemetry.sdk.language","Value":{"Type":"STRING","Value":"go"}},{"Key":"telemetry.sdk.name","Value":{"Type":"STRING","Value":"opentelemetry"}},{"Key":"telemetry.sdk.version","Value":{"Type":"STRING","Value":"1.11.1"}}],"ScopeMetrics":[]}
```
after setting the global meterprovider the result is 
```
{"Resource":[{"Key":"service.name","Value":{"Type":"STRING","Value":"unknown_service:server"}},{"Key":"telemetry.sdk.language","Value":{"Type":"STRING","Value":"go"}},{"Key":"telemetry.sdk.name","Value":{"Type":"STRING","Value":"opentelemetry"}},{"Key":"telemetry.sdk.version","Value":{"Type":"STRING","Value":"1.11.1"}}],"ScopeMetrics":[{"Scope":{"Name":"go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp","Version":"semver:0.36.4","SchemaURL":""},"Metrics":[{"Name":"http.server.request_content_length","Description":"","Unit":"","Data":{"DataPoints":[{"Attributes":[{"Key":"http.flavor","Value":{"Type":"STRING","Value":"1.1"}},{"Key":"http.host","Value":{"Type":"STRING","Value":"http-server:7777"}},{"Key":"http.method","Value":{"Type":"STRING","Value":"GET"}},{"Key":"http.scheme","Value":{"Type":"STRING","Value":"http"}},{"Key":"http.server_name","Value":{"Type":"STRING","Value":"Hello"}}],"StartTime":"2022-11-05T04:39:46.704318664Z","Time":"2022-11-05T04:40:46.669193115Z","Value":0}],"Temporality":"CumulativeTemporality","IsMonotonic":true}},{"Name":"http.server.response_content_length","Description":"","Unit":"","Data":{"DataPoints":[{"Attributes":[{"Key":"http.flavor","Value":{"Type":"STRING","Value":"1.1"}},{"Key":"http.host","Value":{"Type":"STRING","Value":"http-server:7777"}},{"Key":"http.method","Value":{"Type":"STRING","Value":"GET"}},{"Key":"http.scheme","Value":{"Type":"STRING","Value":"http"}},{"Key":"http.server_name","Value":{"Type":"STRING","Value":"Hello"}}],"StartTime":"2022-11-05T04:39:46.704326714Z","Time":"2022-11-05T04:40:46.669195185Z","Value":14}],"Temporality":"CumulativeTemporality","IsMonotonic":true}},{"Name":"http.server.duration","Description":"","Unit":"","Data":{"DataPoints":[{"Attributes":[{"Key":"http.flavor","Value":{"Type":"STRING","Value":"1.1"}},{"Key":"http.host","Value":{"Type":"STRING","Value":"http-server:7777"}},{"Key":"http.method","Value":{"Type":"STRING","Value":"GET"}},{"Key":"http.scheme","Value":{"Type":"STRING","Value":"http"}},{"Key":"http.server_name","Value":{"Type":"STRING","Value":"Hello"}}],"StartTime":"2022-11-05T04:39:46.704333789Z","Time":"2022-11-05T04:40:46.669196421Z","Count":1,"Bounds":[0,5,10,25,50,75,100,250,500,750,1000,2500,5000,7500,10000],"BucketCounts":[0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"Min":0.102534,"Max":0.102534,"Sum":0.102534}],"Temporality":"CumulativeTemporality"}}]}]}
```